### PR TITLE
Added tests for quantize function - Till Saenger

### PR DESCRIPTION
**Pull request by TillRS**

### CODE REVIEW

File: tests/test_quantize.py [hunk 1]
```diff
@@ -1 +1,25 @@
...
+    # Simple two element test
+    assert (quantize(np.array([-1,10.0]), 5) == np.array([0, 4])).all()
+    assert (quantize(np.array([-1,10.0]), 3) == np.array([0, 2])).all()
+
+    # Simple three element test
+    assert (quantize(np.array([-1,-1,10.0]), 2) == np.array([0, 0, 1])).all()
+    assert (quantize(np.array([-1,-1,10.0]), 3) == np.array([0, 0, 2])).all()
+    assert (quantize(np.array([-1,0,10.0]), 4) == np.array([0, 2, 3])).all()
+
+    # Test for output with known data
```
Comment (henryiii) [inline tests/test_quantize.py]:
```suggestion
# Test for output with known data
def test_quantize_random():
```

Let's keep an arrange-act-assert pattern here. No need to make a single function too big.

File: tests/test_quantize.py [hunk 2]
```diff
@@ -1 +1,25 @@
...
+    assert (quantize(np.array([-1,10.0]), 3) == np.array([0, 2])).all()
+
+    # Simple three element test
+    assert (quantize(np.array([-1,-1,10.0]), 2) == np.array([0, 0, 1])).all()
+    assert (quantize(np.array([-1,-1,10.0]), 3) == np.array([0, 0, 2])).all()
+    assert (quantize(np.array([-1,0,10.0]), 4) == np.array([0, 2, 3])).all()
+
+    # Test for output with known data 
+    gen = np.random.default_rng(42)
+    data = gen.random(10)*2-1
```
Comment (henryiii) [inline tests/test_quantize.py]:
```suggestion
    data = gen.uniform(0, 2, size=10)
```

Very minor preference, I think it reads a bit better. (If you do this, the numbers might need to be updated)

**Comment by TillRS** [comment]
I have included your suggested changes and updated the test_quantize_random to work with the uniformly generated random data.
Do I need to submit a new pull request?

**Comment by henryiii** [comment]
No, I can squash and merge this one!

**Review by henryiii — APPROVED** [review]

**Pull request merged**